agent/src/trident.rs

```diff
@@ -1093,53 +1075,6 @@ impl DomainNameListener {
     }
 }
 
-pub enum Components {
```

Comment:
AgentComponents and WatcherComponents are different, can not be directly deleted, AgentComponents than WatcherComponents a lot of processing, directly delete WatcherComponents is not appropriate. It is better to retain WatcherComponents. However, WatcherComponents do not need start() and stop(), and are only used as types to distinguish them and facilitate subsequent expansion